threedi_model_migration/sql.py

```diff
@@ -55,3 +59,17 @@ def select(full_path, query):
         con.close()
 
     return records
+
+
+def filter_global_settings(full_path, settings_id):
+    """Remove global settings, keeping only `settings_id`"""
+    con = sqlite3.connect(full_path)
+    try:
+        with con:
+            con.execute(DELETE_GLOBAL_SETTING.format(settings_id=settings_id))
```

Comment:
Might be vulnerable for SQL injection, but since setings_id is not coming from user data, should be safe.

Reply:
Yes I thoughts so as well. I'll to an "int" cast here just to be sure.